src/questions/expression_control.yaml

```diff
@@ -91,4 +91,576 @@
       key: C
     - content: Address
       key: D
+- uuid: 33765469-a9f8-41d7-903a-3ee4e11f2483
+  type: SingleChoice
+  content: How much parameters are required for defining a for loop in Solidity?
+  hint: noHint
+  explanation: The for loop takes three parameters, one of which is required and the other two are optional. The required parameter must be a conditional statement.
+  answerKeys:
+    - C
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: three
+      key: A
+    - content: four
+      key: B
+    - content: one
+      key: C
+    - content: two
+      key: D
+- uuid: f4cb54b6-26de-4f2b-b879-d621fcd951ea
+  type: SingleChoice
+  content: What is the data type of the required parameter in the for loop?
+  hint: noHint
+  explanation: The required argument should be a conditional statement, hence the data type is boolean.
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: Boolean
+      key: A
+    - content: Integer
+      key: B
+    - content: String
+      key: C
+    - content: Struct
+      key: D
+- uuid: ea3b6f23-b673-4f9a-b928-2bc1ba77ab65
+  type: SingleChoice
+  content: How generally the primary argument withinside the for loop is executed?
+  hint: noHint
+  explanation: The first argument in the for loop is executed only once.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: Infinite
+      key: A
+    - content: One
+      key: B
+    - content: Three
+      key: C
+    - content: It will execute as long as the condition is true
+      key: D
+- uuid: e38441e9-3e06-4910-ba4b-104917d8a641
+  type: SingleChoice
+  content: Is it possible to define single line if/else statement?
+  hint: If/else statments
+  explanation: We can define single if/else statements without curly braces.
+  answerKeys:
+    - C
+  subTopics:
+    - "decision-making"
+  difficultyLevel: Low
+  choices:
+    - content: No it is not possible
+      key: A
+    - content: It is possible only when if/else statements is defined inside function modifiers
+      key: B
+    - content: Yes it is possible
+      key: C
+    - content: None of the above
+      key: D
+
+- uuid: 078cc470-7699-49ac-9122-8b51aa6349ec
+  type: SingleChoice
+  content: How many times the third parameter will execute if the body of for loop is executed n times?
```

Comment:
How many times will the third parameter be execute if the body of for loop is executed n times?
---
Give example also in the question